corehq/apps/reports/standard/deployments.py

```diff
@@ -139,6 +141,22 @@ def rows(self):
             )
         return rows
 
+    @property
+    def export_table(self):
+        def _fmt_ordinal(ordinal):
+            if ordinal >= 0:
```

Comment:
Lol that's pretty interesting. I vote for explicit over implicit

On Monday, July 27, 2015, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/reports/standard/deployments.py
> <https://github.com/dimagi/commcare-hq/pull/7641#discussion_r35554466>:
>
> > @@ -139,6 +141,22 @@ def rows(self):
> >              )
> >          return rows
> >
> > +    @property
> > +    def export_table(self):
> > +        def _fmt_ordinal(ordinal):
> > +            if ordinal >= 0:
>
> (not 100% sure we should be relying on this behavior but it doesn't need
> to block)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/7641/files#r35554466>.
>